cmd/user-agent/src/pages/WebWallet.vue

```diff
@@ -232,6 +234,40 @@ SPDX-License-Identifier: Apache-2.0
                 this.showResp(result.data)
                 this.responses.push("Successfully got DID authorization from wallet.")
             },
+            didConnect: async function () {
+                this.clearResults()
+                if (this.interopData.length == 0) {
+                    this.errors.push("Invalid invitation")
+                    return
+                }
+
+
+                let invitation = JSON.parse(this.interopData)
+                if (invitation['@type'] != 'https://didcomm.org/didexchange/1.0/invitation') {
```

Comment:
this should be an oob invitation message type

Reply:
this is test page code, sometimes people forget to copy invitation in text area and wallet fails. Just check is just to help test page user to if they forget to send invitation.